content/integrations/aws.md

```diff
@@ -56,6 +56,7 @@ There are a number of other AWS services that are also available in Datadog but
 | Lambda |
 | MachineLearning |
 | OpsWorks |
+| Route53 |
```

Comment:
@tmichelet To appear in that list here http://docs.datadoghq.com/integrations/aws/ : 

![screen shot 2016-05-12 at 3 13 01 pm](https://cloud.githubusercontent.com/assets/10196648/15227218/1fa9905c-1854-11e6-9941-8d0df495ec75.png)